Makefile

```diff
@@ -219,34 +184,19 @@ osx: cmake-osx
 	cmake --build ${OSX_BUILD_DIR}
 
 cmake-osx:
-	@mkdir -p ${OSX_BUILD_DIR}
-	@cd ${OSX_BUILD_DIR} && \
-	cmake ../.. ${DARWIN_CMAKE_PARAMS}
-
-OSX_BUILD = \
-	xcodebuild -target ${OSX_TARGET} -project ${OSX_XCODE_BUILD_DIR}/${OSX_XCODE_PROJ} -configuration ${BUILD_TYPE}
+	cmake -H. -B${OSX_BUILD_DIR} ${OSX_CMAKE_PARAMS}
```

Comment:
Nice, did not know about these un documented parameters. (https://stackoverflow.com/a/31092052)